components/history/graphql/card.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div>
+    <div class="show-on-large-screen">
+      <li>
+        <input
+          :aria-label="$t('token_req_name')"
+          type="text"
+          readonly
+          :value="entry.url"
+          :placeholder="$t('empty_req_name')"
+          class="bg-transparent"
+        />
+      </li>
+      <button
+        class="icon"
+        :class="{ stared: entry.star }"
+        @click="$emit('toggle-star')"
+        v-tooltip="{
+          content: !entry.star ? $t('add_star') : $t('remove_star'),
+        }"
+      >
+        <i class="material-icons">
+          {{ entry.star ? "star" : "star_border" }}
+        </i>
+      </button>
+      <button
+        class="icon"
+        @click="expand = !expand"
+        v-tooltip="{
+          content: expand ? 'show less' : 'show more',
+        }"
+      >
+        <i class="material-icons">
+          {{ expand ? "compress" : "expand" }}
+        </i>
+      </button>
+      <v-popover>
+        <button class="tooltip-target icon" v-tooltip="$t('options')">
+          <i class="material-icons">more_vert</i>
+        </button>
+        <template slot="popover">
+          <div>
+            <button
+              class="icon"
+              @click="$emit('use-entry')"
+              :aria-label="$t('edit')"
+              v-close-popover
+            >
+              <i class="material-icons">restore</i>
+              <span>{{ $t("restore") }}</span>
+            </button>
+          </div>
+          <div>
+            <button
+              class="icon"
+              @click="$emit('delete-entry')"
+              :aria-label="$t('delete')"
+              v-close-popover
+            >
+              <i class="material-icons">delete</i>
+              <span>{{ $t("delete") }}</span>
+            </button>
+          </div>
+        </template>
+      </v-popover>
+    </div>
+    <div class="show-on-large-screen">
+      <li>
+        <input
+          v-for="(line, index) in expand
+            ? entry.query.split('\n')
+            : entry.query.split('\n').slice(0, 2).concat(['...'])"
```

Comment:
You can't use expand (`boolean` data type) on `v-for`. `ternary operator` doesn't seems to be working inline inside `v-for`. To iterate over `entry.query`, you can go with an approach similar to this:

```
          v-for="(line, index) in getExpandQueryLines"
```

```
export default {
	  ...
	  ...
	computed: {
		getExpandQueryLines() {
			return this.expand ? entry.query.split('\n') : entry.query.split('\n').slice(0, 2).concat(['...'])
		}
	}
	  ...
	  ...
}
```

Haven't tested this code, please let me know if this fix DeepCode analysis. @oshhh